tests/lib/Event/ContentServiceTest.php

```diff
@@ -1153,6 +1155,93 @@ public function testRevealContentStopPropagationInBeforeEvents()
             [RevealContentEvent::class, 0],
         ]);
     }
+
+    public function testLoadContentEvents(): void
+    {
+        $traceableEventDispatcher = $this->getEventDispatcher(
+            BeforeLoadContentEvent::class,
+            LoadContentEvent::class
+        );
+
+        $content = $this->createMock(Content::class);
+        $innerServiceMock = $this->createMock(ContentServiceInterface::class);
+        $innerServiceMock->method('loadContent')->willReturn($content);
+
+        $service = new ContentService($innerServiceMock, $traceableEventDispatcher);
+        $result = $service->loadContent(2, []);
+
+        $calledListeners = $this->getListenersStack($traceableEventDispatcher->getCalledListeners());
+
+        $this->assertSame($content, $result);
+        $this->assertSame($calledListeners, [
+            [BeforeLoadContentEvent::class, 0],
+            [LoadContentEvent::class, 0],
+        ]);
+        $this->assertSame([], $traceableEventDispatcher->getNotCalledListeners());
+    }
+
+    public function testReturnLoadContentResultInBeforeEvents(): void
+    {
+        $traceableEventDispatcher = $this->getEventDispatcher(
+            BeforeLoadContentEvent::class,
+            LoadContentEvent::class
+        );
+
+        $content = $this->createMock(Content::class);
+        $eventContent = $this->createMock(Content::class);
+        $innerServiceMock = $this->createMock(ContentServiceInterface::class);
+        $innerServiceMock->method('loadContent')->willReturn($content);
+
+        $traceableEventDispatcher->addListener(BeforeLoadContentEvent::class, static function (BeforeLoadContentEvent $event) use ($eventContent) {
+            $event->setContent($eventContent);
+        }, 10);
+
+        $service = new ContentService($innerServiceMock, $traceableEventDispatcher);
+        $result = $service->loadContent(2, []);
+
+        $calledListeners = $this->getListenersStack($traceableEventDispatcher->getCalledListeners());
+
+        $this->assertSame($eventContent, $result);
+        $this->assertSame($calledListeners, [
+            [BeforeLoadContentEvent::class, 10],
+            [BeforeLoadContentEvent::class, 0],
+            [LoadContentEvent::class, 0],
+        ]);
+        $this->assertSame([], $traceableEventDispatcher->getNotCalledListeners());
```

Comment:
Static methods should be called statically.

```suggestion
        self::assertSame($eventContent, $result);
        self::assertSame($calledListeners, [
            [BeforeLoadContentEvent::class, 10],
            [BeforeLoadContentEvent::class, 0],
            [LoadContentEvent::class, 0],
        ]);
        self::assertSame([], $traceableEventDispatcher->getNotCalledListeners());
```